client/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM node:12.14.1-alpine3.11
+RUN apk update
+RUN apk --no-cache add git
+RUN apk --no-cache add nginx
```

I think you can combine these into a single command like this:

```suggestion
RUN apk add -U --no-cache git nginx
```